### PR TITLE
#167484555 User should be able to like and dislike articles

### PR DESCRIPTION
**Pull request by Jkadhuwa**
### What does this PR do?
- Enable authenticated users to like and dislike articles
### Description of Task to be completed?
- Create UI for Viewing single article and like and disliking articles.
- Connect the UI to the backend API
- Test the functionality

### How should this be manually tested?
- Clone this branch
- `npm install` to install all dependencies
- `npm start` to run the server
- `Signup or login to like or dislike articles`
- On the Home page, select any article and proceed to click the like or dislike buttons

### What are the relevant pivotal tracker stories?
- [Like or Dislike Articles](https://www.pivotaltracker.com/story/show/167484555)

### CODE REVIEW

**Review by placideirandora — COMMENTED** [review]
The functionality of liking and disliking an article works as expected.

File: src/feature/articles/getSingleArticle/__tests__/GetSingleArticleComponent.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ViewSingleArticle } from '../GetSingleArticleComponent';
+import LikeDislikeArticle from '../../likeDislikeArticles/LikeDislikeComponent';
+import LocalStorage from '../../../../__mocks__/localStorage';
+
+let storage;
```
Comment (raymond42) [inline src/feature/articles/getSingleArticle/__tests__/GetSingleArticleComponent.test.js]: Can you remove unused imports and declarations from line 5,6 and 8?

**Review by raymond42 — CHANGES_REQUESTED** [review]

File: src/feature/articles/getSingleArticle/__tests__/GetSingleArticleComponent.test.js (hunk 1 above)
Comment (Jkadhuwa) [reply src/feature/articles/getSingleArticle/__tests__/GetSingleArticleComponent.test.js]: Done. Thanks.

**Review by malaba6 — COMMENTED** [review]
Looks good to me

**Comment by malaba6** [comment]
> ### What does this PR do?
> * Enable authenticated users to like and dislike articles
> 
> ### Description of Task to be completed?
> * Create UI for Viewing single article and like and disliking articles.
> * Connect the UI to the backend API
> * Test the functionality
> 
> ### How should this be manually tested?
> * Clone this branch
> * `npm install` to install all dependencies
> * `npm start` to run the server
> * `Signup or login to like or dislike articles`
> * On the Home page, select any article and proceed to click the like or dislike buttons
> 
> ### What are the relevant pivotal tracker stories?
> * [Like or Dislike Articles](https://www.pivotaltracker.com/story/show/167484555)

**Review by malaba6 — APPROVED** [review]

**Review by raymond42 — APPROVED** [review]

**Pull request closed without merging**